fdmprinting/archive_retrieve.py

```diff
@@ -156,7 +156,7 @@ def get_status_from_mqtt(printer, printer_id):
             tls=tls
         )
         printer_object = json.loads(msg.payload)
-     
+        printer_object["print"]["subtask_name"] = printer_object["print"]["subtask_name"].replace("'","").replace('"',"")
```

Comment:
lets try and use `re.sub()` [instead](https://stackoverflow.com/a/60684484) maybe?  I suspect we might find more chars in the near future like line breaks, escape chars etc.  This is user input which we should treat as hostile.

i might even flip this and only allow something like `[a-zA-Z0-9_-\. ]` and call it a day.  We really don't care if the name gets slimmed down and we really don't want it to break :shrug: